components/places/android/src/main/java/org/mozilla/places/LibPlacesFFI.kt

```diff
@@ -61,11 +61,16 @@ internal interface LibPlacesFFI : Library {
             out_err: RustError.ByReference
     ): Pointer?
 
+    /** Note: urls_len and buffer_len must be the same length. The argument is somewhat redundant, but
```

Comment:
You should mention here that these lengths are "how many elements are there", and not byte lengths.